### PR TITLE
Adds section on supported browsers to the MinIO Console page

### LINKED ISSUE #1070: Supported browser list?
Consider adding something to the MinIO Console page about the [supported browers](https://github.com/minio/console/blob/99cf3b378f012b5c53bd9033fdb9c7148e173684/portal-ui/package.json#L46C1-L46C1).

The list of supported browsers is:
- \>0.2% of global users
- Currently supported/live versions
- NOT Opera Mini

The list is built dynamically based on this site:
https://browsersl.ist/#q=%3E0.2%25%2Cnot+dead+and+not+op_mini+all

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #1070

Staged:
- http://192.241.195.202:9000/staging/browsers/linux/administration/minio-console.html#supported-browsers

### CODE REVIEW

**Comment by djwfyi** [comment]
Reference:
https://github.com/minio/console/blob/99cf3b378f012b5c53bd9033fdb9c7148e173684/portal-ui/package.json#L46C1-L46C1

File: source/administration/minio-console.rst [hunk 1]
```diff
@@ -28,6 +28,28 @@ You can use the MinIO Console for administration tasks like Identity and Access
 The MinIO Console is embedded as part of the MinIO Server. 
 You can also deploy a standalone MinIO Console using the instructions in the :minio-git:`github repository <console>`.
 
+Supported Browsers
+~~~~~~~~~~~~~~~~~~
+
+MinIO Console runs on a variety of current, stable release browsers.
+
+MinIO recommends use the latest stable release of your preferred browser for the best experience in the MinIO Console.
```
Comment (feorlen) [inline source/administration/minio-console.rst:36]: typo
```suggestion
MinIO recommends using the latest stable release of your preferred browser for the best experience in the MinIO Console.
```

Comment (feorlen) [reply source/administration/minio-console.rst:36]: Perhaps also switch around the emphasis?
```suggestion
For the best experience in the MinIO Console, MinIO recommends using the latest stable release of your preferred browser.
```

File: source/administration/minio-console.rst [hunk 2]
```diff
@@ -28,6 +28,28 @@ You can use the MinIO Console for administration tasks like Identity and Access
...
+
+- Chrome
+- Edge
+- Safari
+- Firefox
+- Opera
+
+This list is *not* exhaustive and is subject to change.
+
+For a full list of browsers and versions for running MinIO Console, see `Browserslist <https://browsersl.ist/#q=%3E0.2%25%2Cnot+dead+and+not+op_mini+all>`__.
```
Comment (feorlen) [inline source/administration/minio-console.rst:47]: Perhaps not relevant comment: I'm not sure what to make of `Browserslist` as a word. If you don't know what that is (hi :wave:) it looks like a typo.

Maybe use its domain name instead? `Browsersl.ist`? I'm not convinced that is much better, but at least there's a clue this is talking about a website.

**Review by feorlen — APPROVED** [review]
Added a suggestion for a typo (+ alternate text if you like.) Also a 🤔 about `Browserlist` but totally up to you what to do, if anything. Maybe I'm just confused.

Definitely need this info in the docs 💯

File: source/administration/minio-console.rst (hunk 2 above)
Comment (djwfyi) [reply source/administration/minio-console.rst:47]: Trying a different approach.

File: source/administration/minio-console.rst (hunk 1 above)
Comment (djwfyi) [reply source/administration/minio-console.rst:36]: Riffing off your alternative suggestion.

**Review by feorlen — APPROVED** [review]
🏅

**Pull request merged**